src/main/scala/streamit/implicits/implicits.scala

```diff
@@ -0,0 +1,17 @@
+package streamit
+
+import cats.Show
+import streamit.runner.{ Failure, Result, Success }
+
+import scala.language.implicitConversions
+
+package object implicits {
+
+  implicit val showResult: Show[Result] = (result: Result) => {
```

Comment:
I’d recommend no orphaned instances. You can/should move in the `Result` companion object (given it’s an ADT I’d also move the two concrete types in it)

Reply:
Done!